tests/config/functional/ini/pylintrc_with_missing_comma.2.out

```diff
@@ -1,3 +1,3 @@
 ************* Module {abspath}
-{relpath}:1:0: E0012: Bad option value for --disable. Don't recognize message logging-not-lazylogging-format-interpolation. (bad-option-value)
-{relpath}:1:0: E0012: Bad option value for --enable. Don't recognize message locally-disabledsuppressed-message. (bad-option-value)
+{relpath}:1:0: E0012: Bad option value for '--disable'. Don't recognize message 'logging-not-lazylogging-format-interpolation'. (bad-option-value)
```

Comment:
We should probably join the messages with a comma.

Reply:
@Pierre-Sassoulas My bad, I meant the message that we don't recognise 😅 

But the comma you added now is also a good changed imo 👍

Reply:
Oops, I was looking at another test just above it and thought they were connected. This is indeed correct.